From: Markus Koschany <apo@debian.org>
Date: Mon, 8 Aug 2022 18:40:02 +0200
Subject: read write permissions

We need to fix r/w permissions for renpy because we use /usr/share/games/renpy
as project directory. This is a preliminary patch.
---
 renpy/script.py | 15 ---------------
 1 file changed, 15 deletions(-)

--- a/renpy/script.py
+++ b/renpy/script.py
@@ -763,21 +763,6 @@
 
                 pickle_data_after_static_transforms = dumps((data, stmts))
 
-                if not renpy.macapp:
-                    try:
-                        with open(rpycfn, "wb") as f:
-                            self.write_rpyc_header(f)
-                            self.write_rpyc_data(f, 1, pickle_data_before_static_transforms)
-                            self.write_rpyc_data(f, 2, pickle_data_after_static_transforms)
-
-                            with open(fullfn, "rb") as fullf:
-                                rpydigest = hashlib.md5(fullf.read()).digest()
-
-                            self.write_rpyc_md5(f, rpydigest)
-                    except Exception:
-                        import traceback
-                        traceback.print_exc()
-
                 self.loaded_rpy = True
 
             elif fn.endswith(".rpyc") or fn.endswith(".rpymc"):
